### ISSUE TITLE
Error when creating new pantheon app

### ISSUE THREAD

**Issue by rjbain** [kalabox/kalabox-app-pantheon#75]
Hey guys, I'm getting this error when trying to create a new pantheon app using `kbox create pantheon`:

````
$ kbox create pantheon
? Choose a Pantheon account. xxxxx@xxxx.xxx
Your Pantheon session has expired. We need to reauth!
? Pantheon dashboard password (xxxxx@xxxx.xxx) *******
? Pantheon dashboard password (xxxxx@xxxx.xxx) *******
Unhandled rejection TypeError: Cannot read property 'account' of undefined
    at /usr/local/bin/node_modules/kalabox-app-pantheon/vendor/kalabox-app-pantheon/app/plugins/kalabox-plugin-pantheon/lib/client.js.jx:294:23
From previous event:
    at Client.reAuthSession (/usr/local/bin/node_modules/kalabox-app-pantheon/vendor/kalabox-app-pantheon/app/plugins/kalabox-plugin-pantheon/lib/client.js.jx:285:4)
    at Client.__request (/usr/local/bin/node_modules/kalabox-app-pantheon/vendor/kalabox-app-pantheon/app/plugins/kalabox-plugin-pantheon/lib/client.js.jx:352:19)
    at Client.getSites (/usr/local/bin/node_modules/kalabox-app-pantheon/vendor/kalabox-app-pantheon/app/plugins/kalabox-plugin-pantheon/lib/client.js.jx:677:15)
    at /usr/local/bin/node_modules/kalabox-app-pantheon/lib/create.js.jx:237:29
    at process._tickCallback (node.js:812:13)
From previous event:
    at Object.kbox.create.add.option.inquire.choices (/usr/local/bin/node_modules/kalabox-app-pantheon/lib/create.js.jx:236:12)
    at Object.utils.runAsync (/usr/local/bin/node_modules/inquirer/lib/utils/utils.js.jx:28:22)
    at Object.<anonymous> (/usr/local/bin/node_modules/inquirer/lib/utils/utils.js.jx:74:11)
    at Object.utils.runAsync (/usr/local/bin/node_modules/inquirer/lib/utils/utils.js.jx:28:22)
    at /usr/local/bin/node_modules/inquirer/lib/utils/utils.js.jx:50:13
    at tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at setDisposable (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5232:36)
    at AnonymousObservable.s [as _subscribe] (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5249:11)
    at AnonymousObservable.Rx.Observable.observableProto.subscribe.observableProto.forEach (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1975:19)
    at /usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:2448:21
    at tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at setDisposable (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5232:36)
    at AnonymousObservable.s [as _subscribe] (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5249:11)
    at AnonymousObservable.Rx.Observable.observableProto.subscribe.observableProto.forEach (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1975:19)
    at MergeObserver.handleSubscribe (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:3368:28)
    at MergeObserver.onNext (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:3374:16)
    at MergeObserver.tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at AutoDetachObserverPrototype.next (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5273:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1709:35)
    at InnerObserver.onNext (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:4809:14)
    at InnerObserver.tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at AutoDetachObserverPrototype.next (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5273:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1709:35)
    at InnerObserver.onNext (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:3408:91)
    at InnerObserver.tryCatcher (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:566:29)
    at AutoDetachObserverPrototype.next (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:5273:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1709:35)
    at scheduleItem (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:2960:16)
    at Scheduler.scheduleNow [as _schedule] (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:1223:50)
    at Scheduler.Rx.Scheduler.schedulerProto.scheduleWithState (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:969:19)
    at JustSink.run (/usr/local/bin/node_modules/inquirer/node_modules/rx/dist/rx.js.jx:2965:36)


````

**Comment by pirog** [comment]
@rjbain this should be fixed in the 0.11.0 release we are cutting (hopefully) tomorrow!

**Comment by rjbain** [comment]
Thanks, @pirog! Glad they let you outta the code dungeon.
